include/nlohmann/thirdparty/hedley/hedley.hpp

```diff
@@ -504,7 +504,8 @@
 #if \
     defined(__has_cpp_attribute) && \
     defined(__cplusplus) && \
-    (!defined(JSON_HEDLEY_SUNPRO_VERSION) || JSON_HEDLEY_SUNPRO_VERSION_CHECK(5,15,0))
+    (!defined(JSON_HEDLEY_SUNPRO_VERSION) || JSON_HEDLEY_SUNPRO_VERSION_CHECK(5,15,0)) && \
```

Comment:
Why did you change the Hedley header? We copy this from https://github.com/nemequ/hedley, so the change is best made upstream.

Reply:
@nlohmann This is only for intermediate hacking to get it working. I've already opened https://github.com/nemequ/hedley/issues/40.